apps/ledger-live-desktop/src/renderer/components/MainSideBar/index.jsx

```diff
@@ -202,6 +207,24 @@ const TagContainer = ({ collapsed }: { collapsed: boolean }) => {
   ) : null;
 };
 
+const TagContainerFeatureFlags = ({ collapsed }: { collapsed: boolean }) => {
+  const isFeatureFlagsButtonVisible = useSelector(featureFlagsButtonVisibleSelector);
+  const overriddenFeatureFlags = useSelector(overriddenFeatureFlagsSelector);
+
+  const { t } = useTranslation();
+
+  return isFeatureFlagsButtonVisible || Object.keys(overriddenFeatureFlags).length !== 0 ? (
```

Comment:
The `||` here and the description of that setting in the UI "Show banner when flags are overridden" are in conflict in my opinion. It's rather an "Always show FF banner".
I think it's a good idea to have a switch to always display the "banner" (which should be reworded btw), regardless of the state of overriding, but then a different wording in the settings is needed.

Now that this tag/button/banner can be "always displayed", it would be great to include some info in the UI: a simple "overridden"/"no override" would do the job in my opinion.